src/components/common/FullInfoProductChildrenRoute/Feedback/index.tsx

```diff
@@ -0,0 +1,127 @@
+import { FC, useState } from 'react';
+
+import send from '../../../../assets/icons/fullInfoCard/send.svg';
+import Rating from '../../starReting';
+
+const Feedback: FC = () => {
+  const [feedback, setFeedback] = useState<string>('');
+
+  const handleSubmit = (e: React.ChangeEvent<HTMLFormElement>): void => {
+    e.preventDefault();
+    console.log('коментар', feedback);
+    setFeedback('');
+  };
+
+  const arr: number[] = [0, 1, 2, 3, 4, 5];
+
+  return (
+    <div className="w-[851px]   ">
+      <div className=" w-[820px] h-[350px] overflow-y-auto mb-4  ">
+        <ul>
+          {arr.map(com => (
+            <li key={com} className="mb-4 mr-6">
+              <div className="border  border-stroke_input rounded-lg">
+                <div className="flex justify-between px-4 py-2 border-b mb-4 border-stroke_input">
+                  <h2 className="font-medium text-text_com">Фірма “Крона”</h2>
+                  <p className=" text-disabled">17.08.2023</p>
+                </div>
+                <div className="px-4">
+                  <div className="mb-2">
+                    <Rating star={4} active={true} />
+                  </div>
+                  <div>
+                    <p className=" text-text_com">
+                      Надійний покупець. Купував у мене яблуки, груши та
+                      абрикоси домовились що він саме приїде та забере у мене
+                      продукти.
```

Comment:
You should receive reviews from the api and use them on the frontend. So you need to create request to the api or get info with existing one. If api isn't ready, it will still be better to create api request and use answer from there here:
{userReview} or default text for testing UI for now. Later on you will have to delete the default text and show only info from the API. Ideally, the server should provide the necessary test data.

Reply:
This comment applies to all text values, which will be different for each product (entity)